build-npm-module/index310.js

```diff
@@ -0,0 +1,8 @@
+const caseConverter = require('./index');
+
+const str = 'hello free Code Camp!';
+console.log(caseConverter.getUpperCase(str));
+
+// Let's try your getLowerCase function. Conver `str` to lower case and print it.
```

Comment:
Let's try your `getLowerCase()` function. Convert `str` to lower case and print it to the console.